### ISSUE TITLE
Prepoznavanje demencije na osnovu snimaka MRI mozga

### ISSUE THREAD

**Issue by OgnjenFrancuski** [ftn-ai-lab/sc-2017-siit#6]
## Tim
Miloš Pavlić (sw29-2014)
Petar Radošević (sw31-2014) 
Ognjen Francuski (sw8-2014)

## Definicija problema
Prepoznavanje demencije na osnovu snimaka MRI mozga pacijenta.

## Motivacija problema
Rešavanje ovog problema može da pomogne u medicini. Lakše i brže prepoznavanje demencije može dovesti do ranijeg tretmana simptoma kod pacijenata.

## Skup podataka
Podaci su pronađeni na [kaggle-u](https://www.kaggle.com/jboysen/mri-and-alzheimers) koji je referencirao na [oasis-brains.org](http://www.oasis-brains.org/app/template/Tools.vm). Podaci su dati u NIfTI-1 formatu (img - cuva podatke o snimku, hdr - metapodatke). Pomoću [nibabel-a](http://nipy.org/nibabel/) su iz NIfTI-1 podataka dobijeni preseci mozga u tri ose, sačuvani u png formatu.

## Metodologija
- Za realizaciju projekta ce se koristi programski jezik Python, sa bibliotekom Tensorflow
- **CNN** - prepoznavanje klasa demencije na osnovu slika koje predstavljaju MRI snimak mozga
- **ANN** - prepoznavanje klasa demencije na osnovu ulaznih podataka kao što su pol, starost, godište i medicinksih parametara dobijenih testiranjem pacijenata.
- **NEAT** - implementacija NEAT-a (NeuroEvolution of Augmenting Topologies) algoritma koji bi evoluirao neuronske mreže na ovom projektu, a možda i na drugim, kako se ne bi moralo misliti o arhitekturi mreže.

## Model evaluacije
Podaci će biti podeljeni u tri seta podataka:
- trening, 
- test i
- validacioni
Za CNN koristićemo Cross-Entropy za treniranje i za evaluaciju mreža, dok ćemo za ANN koristiti Accuracy.

**Comment by vdragan1993** [comment]
Tema odobrena.
Ostavite link ka GitHub repozitorijumu projekta.
Srećan rad.
